react/typings/vtex.pixel-manager.d.ts

```diff
@@ -0,0 +1 @@
+declare module 'vtex.pixel-manager'
```

Comment:
This should also be unnecessary. Try running `vtex setup --typings` in the root folder for this repo, it should download the correct type definitions for all VTEX IO apps.

Reply:
@victorhmp Fixed typings by adding the dependency on manifest.json and running the command you suggested.